src/index.html

```diff
@@ -7,6 +7,345 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region moyo_header -->
+    <header class="header">
+      <a class="logo" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a href="#" class="menu__link is-active">Apple</a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#samsung" class="menu__link">Samsung</a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__link">Smartphones</a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              href="#laptops&computers"
+              class="menu__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__link">Gadgets</a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__link">Tablets</a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__link">Photo</a>
+          </li>
+
+          <li class="menu__item">
+            <a href="#" class="menu__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <!-- #region card_catalog -->
+    <main class="catalog">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27' Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <span class="card__code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <div class="stars stars--rating">
```

Comment:
Please use stars block from previous tasks. It must have a modifier indicating the number of active stars
```suggestion
          <div class="stars stars--4">
```